login.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="ko">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" as="style" crossorigin
+        href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css" />
+    <link rel="stylesheet" href="/style/reset.css">
+    <link rel="stylesheet" href="/style/base.css">
+    <link rel="stylesheet" href="/style/login.css">
+    <title>로그인</title>
+</head>
+
+<body>
+    <header>
+        <a href="/"><img src="img/logo.png" alt="판다마켓 로고" style="width: 40rem"></a>
```

Comment:
❗️ 수정요청
inline 스타일은 우선순위가 높으니 지양해주세요~
반응형 사이트를 만드실 예정이니 가능하면 css 로 해주세요~

```suggestion
        <a href="/"><img src="img/logo.png" alt="판다마켓 로고" ></a>
```